src/ledc.rs

```diff
@@ -191,6 +191,7 @@ pub struct LedcDriver<'d> {
     hpoint: HPoint,
     speed_mode: SpeedMode,
     max_duty: Duty,
+    pin: i32,
```

Comment:
if we don't make apply config pub, then it can simply accept the pin as an argument.  Both in new() and in set_pin() we have the current pin number.That way we don't need to add a extra pin value, and its less of a breaking change overall.

Reply:
that makes sense, I changed it so there's only an additional pub method, that applies the config with the specified pin